src/context_change.c

```diff
@@ -362,6 +376,9 @@ sr_lycc_del_module(sr_conn_ctx_t *conn, const struct ly_ctx *ly_ctx, const struc
     LY_ERR lyrc;
     int r;
 
+    /* sort removed modules by name because the modules in sr_del_mods are also sorted by name */
+    qsort(mod_set->objs, mod_set->count, sizeof *mod_set->objs, sr_mod_set_qsort_cmp);
```

Comment:
I see no reason for this function to be sorting the modules. If it is required and expected, the set should already be created sorted (the function creating the set should return it ordered).